src/Symfony/Component/Config/Builder/ClassBuilder.php

```diff
@@ -125,9 +126,9 @@ public function addMethod(string $name, string $body, array $params = []): void
         $this->methods[] = new Method(strtr($body, ['NAME' => $this->camelCase($name)] + $params));
     }
 
-    public function addProperty(string $name, string $classType = null): Property
+    public function addProperty(string $name, string $classType = null, bool $normalizeName = true): Property
```

Comment:
Could we easily avoid adding this parameter here?

Reply:
@Nyholm I suppose one way would be to modify the `camelCase` to not trim leading underscores, or maybe check if `$name` starts with an underscore and not call `camelCase` in that case at all.

Reply:
@Nyholm I've implemented the second proposal, if the property name starts with an underscore, `camelCase` is not called. Please let me know if this is ok with you.